templates/accounts.go

```diff
@@ -26,67 +26,118 @@ import (
 )
 
 const createAccountTemplate = `
-transaction(publicKeys: [[UInt8]], code: [UInt8]) {
-  prepare(signer: AuthAccount) {
-	let acct = AuthAccount(payer: signer)
+transaction(publicKeys: [[UInt8]]) {
+	prepare(signer: AuthAccount) {
+		let acct = AuthAccount(payer: signer)
 
-	for key in publicKeys {
-	  acct.addPublicKey(key)
+		for key in publicKeys {
+			acct.addPublicKey(key)
+		}
 	}
-	
-	if code.length > 0 {
-	  acct.setCode(code)
+}
+`
```

Comment:
This template isn't used anymore, right? I would rename the template below to `createAccountTemplate` to avoid confusion, and also to show that it is used by the `CreateAccount` function.

Reply:
sorry didn't catch that wasn't used anymore. I applied the suggested changes.